src/components/picker/picker-component.ts

```diff
@@ -133,7 +133,7 @@ export class PickerCmp {
     this._cols.forEach(column => column.refresh());
   }
 
-  _colChange(selectedOption: PickerColumnOption) {
```

Comment:
revert this, even if the parameter is not used, it is passed by the template. It is good to know which column changed